ppr-ui/src/components/dialogs/StaffPaymentDialog.vue

```diff
@@ -0,0 +1,230 @@
+<template>
+  <v-dialog v-model="setDisplay" width="720px" persistent :attach="attach">
+    <v-card id="staff-payment-dialog" class="pl-5 pr-1 pt-5">
+      <v-row no-gutters>
+        <v-col cols="11">
+          <h3 class="dialog-title pb-6">Staff Payment</h3>
+          <div>
```

Comment:
Hey steph can you please update this to use the base dialog component? You can use slots to replace the body with the staff component / checkbox and props for the title / buttons. You can see I did something similar for the confirmation dialogs. This way all the formatting etc. will be the same / change with anything we alter in the future and we wont have to refactor this with the remaining dialogs

Reply:
done